src/BridgedERC20.sol

```diff
@@ -134,4 +134,16 @@ contract BridgedERC20 is ERC20, AccessControlDefaultAdminRules {
         // Check transfer restrictions
         transferRestrictor.requireNotRestricted(from, to);
     }
+
+    /**
+     * @param account The address of the account
+     * @return Whether the account is blacklisted
+     * @dev Returns true if the account is blacklisted ,if transfer restrictor is not set, if the account is the zero address
+     */
+    function isBlacklisted(address account) external view returns (bool) {
+        if (address(transferRestrictor) == address(0) || account == address(0)) {
```

Comment:
`if (address(transferRestrictor) == address(0)) return false;`

Reply:
This still needs to be fixed

Reply:
so in case there is no transferRestrictor set address(0) would not be blacklisted or do we assume transferRestrictor will always be set?

Reply:
assume not blacklisted if restrictor not set